docs/sdks/python/agents.mdx

```diff
@@ -53,10 +53,10 @@ Furthermore, you can list all existing agents, get agents by name, update agents
 
 ```python
 # list all agents
-agents = agents.list()
+agents = server.agents.list()
 
 # get an agent by name
-agent = agents.get('my_agent')
+agent = server.agents.get('my_agent')
```

Comment:
Thanks @ivancastanop 

Please update also lines 66 and 69 with `server.` in front of `agents.`.

Also, please provide a short video showcasing that this syntax works.